src/components/input/inputDate/inputDate.tsx

```diff
@@ -0,0 +1,41 @@
+import classNames from 'classnames';
+import { forwardRef, useRef } from 'react';
+import { mergeRefs } from 'react-merge-refs';
+import { Button } from '../../button';
+import { IconType } from '../../icon';
+import { InputContainer, type IInputComponentProps } from '../inputContainer';
+import { useInputProps } from '../useInputProps';
+
+export interface IInputDateProps extends IInputComponentProps {}
```

Comment:
`IInputComponentProps` exposes a `maxLength` prop; I'm not sure it makes sense to have that for a date picker. What do you think?

Reply:
Agree, I'm removing it